docs/changelog.rst

```diff
@@ -237,6 +237,8 @@ Other changes:
   :bug:`4585`
 * :doc:`/faq`: :ref:`multidisc`: Elaborated the multi-disc FAQ :bug:`4806`
 * :doc:`/faq`: :ref:`src`: Removed some long lines.
+* :doc:`/guides`: :ref:`main.rst`: Added documentation for updating pip,
```

Comment:
Should be main instead of main.rst but on the other hand: We don't really need a changelog entry for these small docs updates. Just remove the line and we are good :-)